### DISCUSSION TITLE
Docu says Postgres 11 while Requirements suggest latest Postgres

### DISCUSSION THREAD

**Discussion by 4ley** [requarks/wiki#5270] [Help / Questions] [answered]
Reading [the documentation](https://docs.requarks.io/install/ubuntu) it mentions PostgreSQL 11, while [the requirements](https://docs.requarks.io/install/requirements) suggests latest PostgreSQL. Is there any reason to use version 11?

**Comment by NGPixel** [comment, marked as answer]
The docs and examples use 11 because it has been heavily tested. The latest version will however work just fine and you should use it.
